scripts/build_and_test_docker.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+docker run -it --net=host \
+  -v $TRAVIS_BUILD_DIR:/geopyspark \
+  -v $HOME/archives:/root/archives \
+  -v $HOME/.coursier:/root/.coursier \
+  -v $HOME/.ivy2:/root/.ivy2 \
+  -v $HOME/.sbt:/root/.sbt \
+  -e TRAVIS_PYTHON_VERSION=$TRAVIS_PYTHON_VERSION \
+  -e TRAVIS_COMMIT=$TRAVIS_COMMIT \
+  -e COURSIER_PROGRESS=false \
+  -e COURSIER_NO_TERM=true \
+  daunnc/openjdk-gdal:2.4.0 bash -c "cd /geopyspark && ./scripts/run_tests.sh"
```

Comment:
It looks like the script is failing from a permissions error: https://travis-ci.org/locationtech-labs/geopyspark/jobs/491605988#L537

Could you try changing the permissions of `run_tests.sh` so that it can be ran on Travis?